integration/spark/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchemaCommon.scala

```diff
@@ -319,10 +373,6 @@ class AlterTableColumnSchemaGenerator(
         sys.error(s"Duplicate column found with name: $name")
       })
 
-    if (newCols.exists(_.getDataType.isComplexType)) {
```

Comment:
Please update document [Alter-add](https://github.com/apache/carbondata/blob/master/docs/ddl-of-carbondata.md#add-columns)

Reply:
done